com.warlordsoftwares.formatlab.yml

```diff
@@ -0,0 +1,249 @@
+app-id: com.warlordsoftwares.formatlab
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
```

Comment:
do you really need the KDE runtime. You didn't answer the question.

Reply:
yes

Reply:
the binary doesn't link to Qt. So why do you need the KDE runtime?